### DISCUSSION TITLE
How to get the trigger DAG auth on a certain DAG for a user?

### DISCUSSION THREAD

**Discussion by SaturnEver17** [apache/airflow#43497] [Q&A]
In the "Permissions" page, I only found "can read on xx", "can edit on xx" and "can delete on xx", so where is the choice of "can trigger on xx"(xx infers to a certain DAG)?

**Comment by andresitorusz** [comment]
You can check everything related to DAG-level permissions [here](https://airflow.apache.org/docs/apache-airflow-providers-fab/stable/auth-manager/access-control.html#dag-level-permissions).
